scala/StalinSort.scala

```diff
@@ -0,0 +1,33 @@
+import scala.util.Random
+
+object StalinSort {
+  def purge(list: List[Int], prosecutors: Int): List[Int] = {
+    val buf = list.toBuffer
+    buf.remove(prosecutors)
+    buf.toList
+  }
+
+  def stalinSort(list: List[Int]): List[Int] = {
+    var correctList = list
+    var i = 0
+    while (correctList.length > i + 1) {
+      if (correctList(i) > correctList(i + 1)) {
+        correctList = purge(correctList, i + 1)
+      } else {
+        i += 1
+      }
+    }
+
+    correctList
+  }
+}
+
+object Main extends App {
```

Comment:
For pattern matters, could you remove the main function and put an exporting function?

Reply:
i think this is not necessary for now